### DISCUSSION TITLE
Manifest not found

### DISCUSSION THREAD

**Discussion by kostimoshenko** [umputun/remark42#1624] [Q&A] [answered]
Hey guys, 

I do apologise as this seems like an easy issue to solve, but for some reason I can't get my head around it. 

Yesterday I received an email from twitter informing me that my auth app is suspended. I decided to delete this auth method until this issue can be resolved. These are the steps I have taken: 
1. Stopped remark42 with the command docker-compose down 
2. Edited the docker-compose file: deleted twitter auth 
3. Tried to launch remark42 again with default docker-compose pull && docker-compose up -d
4. I receive a following error: 
Pulling remark ... error

ERROR: for remark  b'manifest for umputun/remark42:latest not found'
ERROR: manifest for umputun/remark42:latest not found

I have googled this error and most websites suggest that this must be related to incorrect naming of the image tag in docker-compose file. Here is how my file looks: 

version: '2'

services:
    remark:
        build: .
        image: umputun/remark42:latest
        container_name: "remark42"
        hostname: "remark42"
        restart: always

Please help to understand what has gone wrong and how to fix this

**Comment by paskal** [comment]
Can you please test other images download from that server? It works from my machines, I presume something is off about your docker setup.

**Reply by kostimoshenko** [reply]
Thank you, @paskal 
I am not a developer, so I may have understood your suggestion incorrectly. But I tried this command: 
docker pull hello-world

Here is the output I got: 

> Using default tag: latest
> latest: Pulling from library/hello-world
> 2db29710123e: Pull complete
> Digest: sha256:4e83453afed1b4fa1a3500525091dbfca6ce1e66903fd4c01ff015dbcb1ba33e
> Status: Downloaded newer image for hello-world:latest

Then I did this: docker run hello-world

And the output was: 

> Hello from Docker!
> This message shows that your installation appears to be working correctly.
> 
> To generate this message, Docker took the following steps:
>  1. The Docker client contacted the Docker daemon.
>  2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
>     (amd64)
>  3. The Docker daemon created a new container from that image which runs the
>     executable that produces the output you are currently reading.
>  4. The Docker daemon streamed that output to the Docker client, which sent it
>     to your terminal.
> 
> To try something more ambitious, you can run an Ubuntu container with:
>  $ docker run -it ubuntu bash
> 
> Share images, automate workflows, and more with a free Docker ID:
>  https://hub.docker.com/
> 
> For more examples and ideas, visit:
>  https://docs.docker.com/get-started/

All in all, seems like Docker works

**Reply by paskal** [reply]
And what happens when you run `docker pull umputun/remark42:latest`?

**Reply by kostimoshenko** [reply]
I get this response: 
`Error response from daemon: manifest for umputun/remark42:latest not found`

**Reply by kostimoshenko** [reply, marked as answer]
I think this one is going to help. I tried to pull an older version of remark and took a random version - v1.2.0. And this was successful!
```
v1.2.0: Pulling from umputun/remark42
4fe2ade4980c: Pull complete
d542f82f00ea: Pull complete
5e7deb83e4bf: Pull complete
3e4fa3320f30: Pull complete
299684ba9d65: Pull complete
bb14f103f0f8: Pull complete
983a36b18c83: Pull complete
6a0164896e06: Pull complete
457fec1a6311: Pull complete
35406e2ff6c6: Pull complete
75c3afdbaba1: Pull complete
9e9bf8ad7f2e: Pull complete
a3794fce0c80: Pull complete
ee1a845f219a: Pull complete
Digest: sha256:370742783d93a60e8105a8a4885b412392d9deef1eecacb3b1505f7f64fe9e47
Status: Downloaded newer image for umputun/remark42:v1.2.0 
```

Does this mean that I have to update Docker or something like this?

**Reply by kostimoshenko** [reply]
yes, after I updated Docker to the latest version everything worked like a charm
